python/cudf/cudf/tests/test_orc.py

```diff
@@ -8,6 +8,7 @@
 import pandas as pd
 import pyarrow as pa
 import pyarrow.orc
+import pyorc
```

Comment:
CI fails because of this import
```
ModuleNotFoundError: No module named 'pyorc'
```

Reply:
Created a PR for it https://github.com/rapidsai/integration/pull/215